tools/shared.py

```diff
@@ -1873,10 +1873,11 @@ def llvm_opt(filename, opts, out=None):
 
     logging.debug('emcc: LLVM opts: ' + ' '.join(opts) + '  [num inputs: ' + str(len(inputs)) + ']')
     target = out or (filename + '.opt.bc')
-    proc = run_process([LLVM_OPT] + inputs + opts + ['-o', target], stdout=PIPE, check=False)
-    output = proc.stdout
-    if proc.returncode != 0 or not os.path.exists(target):
-      logging.error('Failed to run llvm optimizations: ' + output)
+    try:
+      run_process([LLVM_OPT] + inputs + opts + ['-o', target], stdout=PIPE)
+      assert os.path.exists(target), 'llvm optimizer emitted no output.'
+    except CalledProcessError as e:
```

Comment:
doesn't this need `subprocess.CalledProcessError`?

Reply:
Oops, you're right.